lego/apps/lending/serializers.py

```
@@ -9,6 +9,8 @@
     LENDING_REQUEST_TRANSLATION_MAP,
 )
 from lego.apps.lending.models import LendableObject, LendingRequest
+from lego.apps.users.fields import AbakusGroupField
+from lego.apps.users.models import AbakusGroup
```

Ruff (0.8.2)

`lego.apps.users.models.AbakusGroup` imported but unused

Remove unused import: `lego.apps.users.models.AbakusGroup`

(F401)